src/api.js

```diff
@@ -82,8 +94,8 @@ module.exports = function(ctx) {
         features: ctx.store.getAll().map(feature => feature.toGeoJSON())
       };
     },
-    delete: function(id) {
-      ctx.store.delete([id], { silent: true });
+    delete: function(featureIds) {
```

Comment:
It is actually not breaking: it will accept either a single ID, as before, or an array. But, yes, we need to clean up docs with these changes.